### PR TITLE
Implement Db models

### LINKED ISSUE #17: Events Data Model
## Task(s)
- [ ] Fields: name, link, start/end time, points, secret code, description

### PR DESCRIPTION
**Pull request by luiszugasti**
## Description

Created the schema file for usage with the database. This schema *has not yet been tested* in Postgres.

Fixes #17 and #18

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

## Screenshots
N/A

### CODE REVIEW

**Comment by luiszugasti** [comment]
Quick mockup showing the general relationships between the Entities:
image

**Comment by luiszugasti** [comment]
@cqvu Now the PR should help fix #16

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (cqvu) [inline .gitignore:1]: There should already be a .env file in the backend directory. I don't think we need one in the root dir as well

File: backend/schema.sql [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+*   Assignee has relation to User
+*   Event has relation to Event
+*   TimeStamp is time when secret code was entered. Useful for audit.
+*
+*/
+
+CREATE TABLE Points (
+    InstanceID INTEGER,
+    Assignee NOT NULL,
+    Event NOT NULL,
```
Comment (cqvu) [inline backend/schema.sql:59]: Some points might not have an Event right? i.e Show and Tells, Discord activity points

File: backend/schema.sql [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+/** Users
+*   
+*
+*/
+
+CREATE TABLE Users (
+    UserID INTEGER,
+    DisplayName VARCHAR(255) NOT NULL,
+    Role roles NOT NULL,
+    Pod INTEGER NOT NULL,
```
Comment (cqvu) [inline backend/schema.sql:25]: Would this be VARCHAR? I'm thinking the Pod number would be stored, so something like '2.0.2' which idk if that would work with INTEGER

Comment (luiszugasti) [reply backend/schema.sql:25]: I think you are right that it has to be a var char, overlooked by me!

File: .gitignore (hunk 1 above)
Comment (luiszugasti) [reply .gitignore:1]: Mb I overlooked the .env listing in the gitignore located in "backend" folder

Comment (luiszugasti) [reply .gitignore:1]: PR was resolved by implementing the same data model in models.py

File: backend/schema.sql (hunk 2 above)
Comment (luiszugasti) [reply backend/schema.sql:59]: PR was resolved by implementing the same data model in models.py

File: backend/schema.sql (hunk 3 above)
Comment (luiszugasti) [reply backend/schema.sql:25]: PR was resolved by implementing the same data model in models.py

**Comment by luiszugasti** [comment]
PR was resolved by implementing the same data model in models.py

**Pull request closed without merging**